packages/react-native/src/primitives/RadioGroup/RadioGroup.tsx

```diff
@@ -1,34 +1,50 @@
 import React, { Children, cloneElement, isValidElement, useMemo } from 'react';
 import { View, ViewStyle } from 'react-native';
 
-import { Label } from '@aws-amplify/ui-react-native/dist/primitives';
-import { RadioProps } from '@aws-amplify/ui-react-native/dist/primitives';
+import { Label } from '../Label';
+import { getFlexDirectionFromLabelPosition } from '../Label/utils';
+import { RadioProps } from '../Radio';
 
+import { styles } from './styles';
 import { RadioGroupProps } from './types';
 
 export default function RadioGroup<T>({
+  accessibilityRole = 'radiogroup',
   children,
   direction = 'vertical',
   disabled,
   label,
+  labelPosition = 'top',
   labelStyle,
   onChange,
+  size,
   style,
   value,
   ...rest
 }: RadioGroupProps<T>): JSX.Element {
+  const containerStyle: ViewStyle = useMemo(
+    () => ({
+      style,
```

Comment:
Does this need to be spread?

Reply:
When I spread it, TypeScript throws an error: 

```
Spread types may only be created from object types.ts(2698)
```

Reply:
Ahhh, that's because `ViewProps['style']` can resolve to non-object types. So if i pass a style prop to a `RadioGroup`, does it get applied?

Reply:
Hmm okay, doesn't look like the `style` prop gets applied. I'll look into it some more